### ISSUE TITLE
set video monetization

### ISSUE THREAD

**Issue by realloro** [tokland/youtube-upload#199]
can i set monetize to the video uploaded?

**Comment by Utty88** [comment]
No, that is not possible as the API does not support that any more.

Gesendet von meinem Windows 10 Phone

Von: realloro<mailto:notifications@github.com>
Gesendet: Sonntag, 24. September 2017 18:27
An: tokland/youtube-upload<mailto:youtube-upload@noreply.github.com>
Cc: Subscribed<mailto:subscribed@noreply.github.com>
Betreff: [tokland/youtube-upload] set video monetization (#199)


can i set monetize to the video uploaded?

**Comment by xueenda** [comment]
https://developers.google.com/youtube/partner/upload_claim_sample

policyId: The monetization policy to apply to the uploaded video. The policy must be associated with the YouTube Content Manager account of the authenticated user. The default is the standard YouTube "monetize" policy.

Example: --policyId="S309961703555739"

**Comment by RoelVdP** [comment]
Hi! Can this be implemented please?
